packages/typespec-go/src/tcgcadapter/types.ts

```diff
@@ -161,15 +161,6 @@ export class typeAdapter {
         types.set(m4.SchemaType.Binary, binaryType);
         return binaryType;
       }*/
```

Comment:
Is there a reason to leave the commented code above? Is that a tcgc bug that when fixed we will enable?

Reply:
Streaming content still needs to be supported. It's WIP with the changes for multipart/form data. Will remove with those changes.